app/components/Chart/ProfileChart.js

```diff
@@ -0,0 +1,145 @@
+﻿import React, { Component } from 'react';
+import {Link} from 'react-router'
+
+var LineChart = require("react-chartjs").Line;
+var PieChart = require("react-chartjs").Pie;
```

Comment:
When you install these, make sure you add "--save" at the end of your command. That way it gets added to the packages.json